plugins/ui-update/__tests__/notifications-test.ts

```diff
@@ -0,0 +1,243 @@
+import { BehaviorSubject, of } from "rxjs";
+import { marbles } from "rxjs-marbles/jest";
+
+let mockMetadata$ = of({});
+let mockAction$: BehaviorSubject<any>;
+jest.mock("../streams", () => ({
+  getAction$: () => mockAction$,
+  getUiMetadata$: () => mockMetadata$
+}));
+
+import { ToastNotification } from "@extension-kid/toast-notifications";
+
+import * as Notifications from "../notifications";
+import {
+  EMPTY_ACTION,
+  UIAction,
+  UIActions,
+  UIActionType
+} from "../types/UIAction";
+
+mockAction$ = new BehaviorSubject<UIAction>(EMPTY_ACTION);
+const mockNS = {
+  push: jest.fn()
+};
+
+describe("Notifications", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+  afterEach(() => {
+    Notifications.clearActiveNotifications();
+  });
+  describe("UIUpdatedNotification", () => {
+    it(
+      "pushes notification when client build !== server build",
+      marbles(m => {
+        mockMetadata$ = m.cold("--j", {
+          j: {
+            clientBuild: "master+v1.0.1",
+            packageVersion: "1.2.0",
+            packageVersionIsDefault: false,
+            serverBuild: "master+v1.2.0"
+          }
+        });
+        //@ts-ignore
+        Notifications.setupUIUpdatedNotification(mockNS);
```

Comment:
then here you can create an empty container and bind the mock to it instead of the Notification service